psi4/src/psi4/libfock/jk.cc

```diff
@@ -537,6 +550,88 @@ void JK::AO2USO() {
 }
 void JK::initialize() { preiterations(); }
 
+void JK::incfock_setup() {
```

Comment:
It might be better to have JK::incfock_setup() and JK::incfock_postiter() throw a PSIEXCEPTION since incfock is basically meaningless outside of DirectJK/CompositeJK, then implement the current definition of these functions into DirectJK and CompositeJK. On one hand, it's a bit of code duplication, so that's unfortunate. On the other hand, it clearly delineates which JK subclasses are meant to use these functions.

Reply:
Hm, I am __strongly__ against the code duplication. There is currently __NO__ way incfock is called in non-Direct and non-Composite codes (i.e. those functions are called within another function in DirectJK and CompositeJK)